build/build.mk

```diff
@@ -26,14 +32,26 @@ DOCKER_THRIFT=$(DOCKER_BIN) run --rm -t \
 					--name "nrjmx-code-generator" \
 					-v $(CURDIR):/src/nrjmx \
 					-w /src/nrjmx \
-					nrjmx-code-generator
+					ohaiops/nrjmx-code-generator:latest
 
 .PHONY : code-gen-utils
 code-gen-utils:
-		@($(DOCKER_BIN) build -t nrjmx-code-generator ./commons/.)
+		@($(DOCKER_BIN) build \
```

Comment:
I would expect here that we use image that we published. The publish idea is to avoid building the image every time in the pipeline

Reply:
This is the command to make the image and it is the dependency of `docker/publish`

The target to use the published image is `code-gen` that uses `DOCKER_THRIFT` that is the published image.